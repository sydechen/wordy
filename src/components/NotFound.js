import React from 'react';
import { Link } from 'react-router-dom';


const NotFound = () => (
    <div>
        404..you wanna go <Link to='/'>home?</Link>
    </div>
);

export default NotFound;