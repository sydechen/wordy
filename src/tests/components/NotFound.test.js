import React from 'react';
import NotFound from '../../components/NotFound';
import {shallow} from 'enzyme';

test('render NotFound page', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
})