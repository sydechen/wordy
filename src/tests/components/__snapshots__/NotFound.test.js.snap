// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NotFound page 1`] = `
<div>
  404..you wanna go 
  <[object Object]
    to="/"
  >
    home?
  </[object Object]>
</div>
`;
